etcdctl/ctlv3/command/get_command.go

```diff
@@ -47,6 +48,7 @@ func NewGetCommand() *cobra.Command {
 	cmd.Flags().BoolVar(&getPrefix, "prefix", false, "get keys with matching prefix")
 	cmd.Flags().BoolVar(&getFromKey, "from-key", false, "get keys that are greater than or equal to the given key")
 	cmd.Flags().Int64Var(&getRev, "rev", 0, "specify the kv revision")
+	cmd.Flags().BoolVar(&getKeysOnly, "keys-only", false, "get only the keys (valid only with --prefix)")
```

Comment:
the implementation actually works without --prefix, and i feel it is correct. no?